docs/nnfw/howto/CrossBuildForAndroid.md

```diff
@@ -37,5 +37,5 @@ Here is an example of using Makefile.
 TARGET_OS=android \
 CROSS_BUILD=1 \
 NDK_DIR=/path/android-sdk/ndk/{ndk-version}/ \
-make -f Makefile.template install
+make -f Makefile.template
```

Comment:
@hseok-oh Is it intentionally included in this PR? It is about doc, not infra or Makefile.